Docs matthias v2

bluedeepimpact opened the pull request:
Pull request of the docs changes for 11.0
RWagner set as reviewer.

bluedeepimpact committed:
Create README.md

new subsection for automatic and manual scheduling
[ci skip]

bluedeepimpact committed:
changes in Gantt docs

[ci skip]

bluedeepimpact committed:
Merge branch 'release/11.0' into docs-matthias-v2

bluedeepimpact committed:
Update README.md

bluedeepimpact committed:
Added sub-section for automatic & manual scheduling mode

[ci skip]

bluedeepimpact committed:
Gantt, manual scheduling, work packages

Improved docs for Gantt; prolonged docs for scheduling modes; added table of contents for chapter about work package relations
[ci skip]

bluedeepimpact committed:
minimal changes

[ci skip]

bluedeepimpact committed:
documentation for additional options for project overview list

documentation for Epic #34167 Basic portfoliomanagement for project list

[ci skip]

bluedeepimpact committed:
documentation changes for epic #32125 (time and costs module)

[ci skip]

bluedeepimpact committed:
small additional changes for time and costs module

[ci skip]

bluedeepimpact committed:
Docs changes for new Action boards

[ci skip]

bluedeepimpact committed:
renaming of 'budgets' to 'time and costs' in system admin docs

and additional smaller improvements
[ci skip]

bluedeepimpact committed:
restructured the system admin guide

[ci skip]

RobinWagner commented:
Hi @ML-OpenP,

Thanks a lot for adding these entries to the documentation.
It looks very good. I noticed some areas where I would suggest some changes and corrections.
The following includes some suggested changes to pages that you changed which are unrelated to changes you made - but where I think it would be good to make some changes:

## Gantt Chart (gantt-chart):

**How to change the order of an item in the Gantt chart?**

- “To change the order of the elements in the Gantt chart, click the drag & drop …" (no separate sentences)

**Relations in the Gantt chart**

- “To add a dependency make right mouse click on an element in the Gantt chart.” – change to e.g. “To add a dependency, right-click on an element in the Gantt chart.”

- “Select the item which you want to create the dependency with.” – change to e.g. “Select the item to which you want to create a dependency.”

**Gantt chart configuration**

- “Click the Apply button to save your changes.()” – parentheses can be removed

**Synchronize data from OpenProject to Excel** -> Change to e.g. “Synchronize data between OpenProject and Excel” (or similar – it goes both ways and the way from Excel to OP is more important)

- “You can synchronize your work packages data from OpenProject to Excel (and even convert from or to MS Project). You need to download an OpenProject plugin.” – change to e.g. “You can synchronize your work package data between OpenProject and Excel (two-way synchronization). By extension you could import MS Project files – by saving them as Excel files and importing them into OpenProject. You can download a free Excel synchronization template to do this.”

**Zen Mode** --> we should either remove this or add the documentation (currently there is just headline without any content)


## Automatic and manual scheduling mode (gantt-chart/scheduling)

- To schedule work packages in the Gantt chart there is a**n** **automatic scheduling mode (default)** and a **manual scheduling mode** (new in [release 11.0](../../../release-notes/11-0-0)). 

**Automatic scheduling mode**

- “When setting a dependency between two work packages the automatic scheduling mode is set as default.” – this is independent of setting a dependency. The automatic scheduling mode is generally set as the default mode for new work packages

- “Now work package one gets delayed by ten days, you have to adjust the planning.” – I would write “work package 1” (instead of “one”) to be consistent

**Manual scheduling mode**

- “you want to set a parent work package’s date date independently …” (add apostrophe)

- “you don’t want a parent work package’s date being updated automatically …” (add apostrophe)

- “you don't want a follower's start date to be automatically updated when you change the predecessors finish date”

- I would add that the manual scheduling mode is mostly used when creating a top-down project plane (by defining the main phases first). You can then add child work packages without affecting the original (high-level) project schedule. Ideally we would add that at the beginning of the manual scheduling mode section.


## Manage project information (projects/project-settings/project-information)

- The image seems to still be outdated (does not include “Set as template” button)


## Projects (getting-started/projects)

**Create a new project**

- “You can either create a completely new project, create a subproject of an existing project or create a (sub)project from a template. For the latter option, choose a template using the drop-down menu.” – I would add a link / some information to how users can define templates


## Work package relations and hierarchies (work-packages/work-package-relations-hierarchies)

**Table of contents**

- “Where can I find out about a work package’s children an**d** parent?”

**Adding a child work package**

- “There are three ways to add a child work package” (instead of “create” since e.g. the indent method does not create a new work package but adds an existing work package as a child)

- 1. “Adding **or creating** a child in the Relations tab in a work package’s detail view” (since it could be both)

Otherwise it looks really good.
Let me know if you have any questions regarding the comments.

Thanks a lot!

bluedeepimpact committed:
Documentation for Attribute help texts

[ci skip]

bluedeepimpact committed:
documentation for webhooks on projects

[ci skip]

bluedeepimpact committed:
Doc changes mainly for FR #33580

[ci skip]

bluedeepimpact committed:
Gantt chart, System settings, Copy projects

- Improvements in Gantt chart description (FR #32881)
- changed screenshot for system settings (FR #32051)
- added info about possibility to copy boards and dashboards (FR # 30585)
[ci skip]

RobinWagner commented on docs/user-guide/agile-boards/README.md:
```diff
@@ -39,22 +41,44 @@ The new Boards are tightly integrated with all other project management function
 
-You can create as many flexible boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
-If you haven't done so yet, [activate the Boards module](../projects/project-settings/modules) within your project. Also, we recommend to verify [Roles and Permissions](https://www.openproject.org/help/administration/manage-roles-permissions/) within your system's Administration.
+You can create as many Agile Boards in a project as you need and configure them to your needs. First, you need to create a new Boards view. 
```
We should ideally use one way of capitalizing agile boards (either Agile boards (as in the beginning of the page) or Agile Boards (capitalized)).

RobinWagner commented on docs/user-guide/agile-boards/README.md:
```diff
@@ -39,22 +41,44 @@ The new Boards are tightly integrated with all other project management function
+
+There are several **types of Action boards** available:
+
+**Status board:** Each list represents a status. That means that e.g. all work packages with the status new will be automatically displayed in the column "New". 
```
I would add quotation marks around status "new" to make it a bit clearer what is meant.

RobinWagner reviewed (approved):
Thank you @ML-OpenP.
That looks really good. I just added a couple of comments.

RobinWagner commented on docs/system-admin-guide/README.md:
```diff
@@ -16,15 +16,18 @@ If you are an administrator on your OpenProject environment,  you can access the
 | [Users and permissions](./users-permissions)       | How to manage users, groups and permissions in OpenProject?  |
 | [Work packages](./manage-work-packages)            | How to configure work packages, types, status and workflows? |
 | [Custom fields](./custom-fields)                   | Set custom fields for work packages, Spent time, projects, versions, users, groups and more. |
+| [Attribute help texts](/attribute-help-texts)      | Add help texts to explain custom fields in projects and work packages. |
```
I would change this to e.g. "explain attributes, including custom fields" (since attribute help texts are also available for standard attributes).

RobinWagner reviewed (approved):
Looks good. I just added on comment / suggested change.

RobinWagner reviewed (approved):
Looks good @ML-OpenP. Thank you!

RobinWagner commented on docs/enterprise-edition-guide/README.md:
```diff
@@ -42,5 +42,5 @@ To upgrade your Community installation, navigate to your system -> Administratio
 ### Do you have a reseller program for OpenProject?
 
-We provide a [reseller program](https://www.openproject.org/reseller-program/) exclusively for the OpenProject Enterprise Edition (currently not for the Cloud Edition) and offer a 25% discount on the regular prices as a part of this. Once you purchased the first Enterprise Edition license for a client, you receive an Enterprise Edition license for 25 users free of charge (for internal use).
+We provide a [reseller program](https://www.openproject.org/reseller-program/) exclusively for the OpenProject Enterprise Edition (currently not for the Cloud Edition) and offer a 25% discount on the regular prices as a part of this. Once you purchased the first Enterprise Edition license for a client, you receive an Enterprise Edition license for 25 users free of charge for your internal use. Pl ...
```
We could change "Enterprise Edition" to "Enterprise on-premises" to be consistent with the new naming convention - and "Cloud Edition" to "Cloud on-premises".

RobinWagner reviewed (approved):
Looks good, just added one comment.

RobinWagner reviewed (approved):
Looks good! Thank you!

RobinWagner reviewed (approved):
Looks really good. Thank you.

RobinWagner commented on docs/user-guide/projects/README.md:
```diff
@@ -152,15 +152,20 @@ Another way for using a template project would be to [copy it](#copy-a-project).
-Then click the blue **Copy** button.
+![copy-project-screen-11.0](image-20201007172029375.png)
+
+For further configuration open the **Advanced settings**. Here you can specify (among other things) the project's URL (identifier), its visibility and status. Furthermore you can set values for custom fields (not shown in the screenshot).
```
"it's visibility and status" instead of "its visibility and status"

RobinWagner reviewed (approved):
Looks good. Thank you!

bluedeepimpact committed:
small changes on the basis of the review

[ci skip]

bluedeepimpact reviewed (commented):
I reviewed my changes, should be fine to merge

bluedeepimpact committed:
typo

[ci skip]

bluedeepimpact committed:
Merge branch 'release/11.0' into docs-matthias-v2

bluedeepimpact committed:
embedded video for manual scheduling mode

[ci skip]

ulferts commented:
Merging on the basis of @RobinWagner 's approval.
